lib/__mocks__/requireResolve.js

```diff
@@ -1,7 +1,7 @@
-let resolvedPaths;
+let resolvedPaths = {};
 
-function __setResolvedPaths(paths: object) {
-  resolvedPaths = paths;
+function __addResolvedPath(alias: string, path: string) {
```

Comment:
I think this is fine for now, because of the way our Importer test is structured, but in the long run having to trace up every level to get enough context to understand a given test makes the tests a lot less readable--I think we should try to keep test setup more localized going forward.